src/redux/wallets/state.ts

```diff
@@ -62,6 +62,7 @@ export interface ITokenState {
         error: any;
         available: string;
         total: string;
+        detailed: any;
```

Comment:
Is is possible to add here a type?

Check `export interface IBlockchainTransaction<IAdditionalInfoType = any> {}`, maybe we can use something like this. To have easier tracking of detailed field.

Reply:
It would be nice, but a bit hard in this context... it could just complicate things :)